_posts/en/newsletters/2020-07-08-newsletter.md

```diff
@@ -0,0 +1,227 @@
+  previous transactions were only included for legacy (non-segwit)
+  UTXOs.  This change is a response to some hardware wallets now
+  requiring or recommending access to previous transactions for segwit
+  UTXOs in order to mitigate the [fee overpayment attack][].  If segwit
+  v1 ([taproot][topic taproot]) is adopted, transactions spending all
+  taproot inputs should not need this extra data by default.
+
+- [C-Lightning #3775][] adds RPC methods for [PSBT][topic psbt] lifecycle
+  management backed by C-Lightning's internal wallet. The `reserveinputs` RPC
+  method creates a PSBT by choosing UTXOs from the internal wallet as inputs to
+  satisfy the user-specified list of outputs, marking chosen UTXOs as reserved.
+  The resulting PSBT may either be supplied to the `unreserveinputs` RPC method
+  to manually release the reserved UTXOs, or to the `signpsbt` RPC method to add
+  signatures from the internal wallet. Finally, the `sendpsbt` RPC method will
+  convert a fully-signed PSBT into a ready-to-broadcast transaction, and
+  broadcast it to the network. Users should note that restarting C-Lightning
+  effectively un-reserves all previously reserved UTXOs, requiring a new PSBT be
+  created with `reserveinputs` before `signpsbt` will accept it.
+
+- [Eclair #1427][] and [#1439][Eclair #1439] add support to Eclair for
+  effective sending of [multipath payments][topic multipath
+  payments]---payments which are split into several parts, with each
+  part routed using a different path.  These PRs split payments into up
+  to six parts by default, initially allocating 0.00015 BTC to each part
+  but increasing the value of each part by a semi-random amount until
+  the full payment amount has been allocated.  Once the amounts have all
+  been selected, all of the payments parts are sent.  This is efficient but,
+  perhaps just as importantly, the opportunistic value increasing
+  function may also help prevent any nodes that see a subset of the
+  payments from guessing the full payment amount, improving privacy.  If
+  you're interested in details, both Eclair's and C-Lightning's
+  splitting algorithms were [discussed][split algos] by their authors
+  this week.
+
+- [Eclair #1249][] adds optional support for using PostgreSQL as the
+  database backend instead of the default SQLite.  For details, see
+  Eclair's new [PostgreSQL documentation][eclair postgresql].  See also
+  the report for Optech about [using Eclair in a production
+  environment][eclair production] with PostgreSQL, written by Roman
+  Taranchenko---who is also the author of this PR.
+
+- [Eclair #1473][] updates Eclair's code for handling a mismatch between
+  the onchain feerate selected by a remote channel peer and the feerate
+  the local node thinks is appropriate.  After this change, if the
+  remote peer selects a feerate that seems to be too high, the local
+  node won't close the channel unless it's more than eight times the
```

Comment:
s/eight/10/?
```
    feerate-tolerance {
      ratio-low = 0.5 // will allow remote fee rates as low as half our local feerate
      ratio-high = 10.0 // will allow remote fee rates as high as 10 times our local feerate
    }
```

Reply:
per https://github.com/ACINQ/eclair/pull/1473/files#diff-9dd7d738eab09437470b94582ebe2b51R104

Reply:
Ah, they tricked me!  It's 8.0 in the first commit https://github.com/ACINQ/eclair/commit/8e7f4916ce2be3bf2273f381b4aed3d1aae6dd98#diff-9dd7d738eab09437470b94582ebe2b51R104 but they updated it to 10 in a later commit https://github.com/ACINQ/eclair/pull/1473/commits/bf2000fc9a377852c203da4e439a62fc9c14fe92

Thanks for the catch!